internal/db/schema/migrations/postgres/3/01_auth.up.sql

```diff
@@ -0,0 +1,10 @@
+begin;
+
+
+-- this constraint is intended to ensure that a user cannot have more than one
+-- account per auth_method
+alter table auth_account
```

Comment:
@jefferai @mgaffney and I discussed this last week (sorry we didn't include you).  We aligned on this as an acceptable migration experience.

Reply:
I think the error will return this file so maybe having the comment explain this and maybe how to handle any migration failures as a result of this might be good.  Feel free to ignore as well if the content of the file is considered generally good enough as an explanation of the error and how to fix it.

Reply:
added a comment with a query to help the operator.